lektor_s3.py

```diff
@@ -33,6 +36,16 @@ def __init__(self, env, output_path):
         self.bucket = None
         self.key_prefix = ''
         self.cloudfront = None
+        self.config = self.get_config()
+
+    @property
+    def config_filename(self):
+        return os.path.join(self.env.root_path, 'configs', 's3.ini')
+
+    def get_config(self):
```

Comment:
Oh, you are quite right!

Let's keep it the way you have it. I'll file a ticket on the lektor project that this is too hard right now.